storehaus-elasticsearch/src/main/scala/com/twitter/storehaus/elasticsearch/ElasticSearchStringStore.scala

```diff
@@ -120,18 +116,19 @@ class ElasticSearchStringStore(private val index: String,
   }
 
 
-  def queryable: ReadableStore[SearchRequest, Seq[String]] = new ReadableStore[SearchRequest, Seq[String]] {
-    /** get a single key from the store.
-      * Prefer multiGet if you are getting more than one key at a time
-      */
-    override def get(k: SearchRequest): Future[Option[Seq[String]]] = futurePool {
-      //Force to use the index and Type this store is configured for.
-      val updatedQuery = k.indices(Array(index): _*).types(Array(tipe): _*)
-      val searchHits = client.search(updatedQuery).actionGet().getHits
-      searchHits.totalHits() match {
-        case 0 => None
-        case _ => Some(searchHits.hits().toList.map(_.getSourceAsString))
+  def queryable: ReadableStore[SearchRequest, Seq[String]] =
+    new ReadableStore[SearchRequest, Seq[String]] {
+      /** get a single key from the store.
+       * Prefer multiGet if you are getting more than one key at a time
```

Comment:
this comment is a lie. There is no optimization of multiGet here, right? delete?